### ISSUE TITLE
[BUG] Installation setup stuck

### ISSUE THREAD

**Issue by spacecake** [Joly0/Run-in-Sandbox#5]
Hello Joly,

Unlocking all the files:
`dir "C:\\Run-in-Sandbox-master\Sources\Run_in_Sandbox" -Recurse | Unblock-File`

Running as admin:
![image](https://github.com/Joly0/Run-in-Sandbox/assets/961558/fa0d359e-13c4-4d8a-87bd-0535bf4d79fd)

System: Latest W11 Ent 22H2

any idea?

**Comment by Joly0** [comment]
Hey, i cannot replicate this. Could you please describe your exact steps you did, to get to this error?

**Comment by Joly0** [comment]
Ahh, never mind, i think i might know what it is. Sooo, the next step after the last step in your screenshot `SUCCESS : Directory has been exported` is taking a system restore point. The issue is, this process happens in the background, and can take some time. Thats probably why you think its stuck, while in reality its creating the restore point.

I will probably change it, so you can actually see some progress here, but if you give it a while, it "should" (in theory) continue after a while, however the restore point takes to create.

If you want, i can keep this issue open, until i have changed the script,so it better shows the progress and you can test it :)
Maybe will ask for feedback :)

**Comment by spacecake** [comment]
Left it for 4h, did not make a difference. It might be specific to my system, who knows..
It might a good idea to echo a description for this step in the console, so its clear which step is taking long.

To came out of this, is it possible for you to kindly point me to the code I would need to comment out to skip this step?

**Comment by Joly0** [comment]
You dont need to comment something out. There is a parameter you can set when running the add_structure.ps1 script. The parameter is `-NoCheckpoint`

**Comment by Joly0** [comment]
Hey @spacecake not sure if the parameter helped you, but i just pushed a new update, that should solve your issue once and for all. You can test it, but for the time being, i will close this issue. Feel free to write here again or open a new issue, if you face this issue again :)
